backend/utils/validation.ts

```diff
@@ -0,0 +1,30 @@
+
+
+import {Request, Response, NextFunction} from "express";
+
+import  {validationResult} from "express-validator";
+
+
+export function handleValidationErrors(req: Request,  res: Response, next: NextFunction){
+
+
+    const validationErrors = validationResult(req);
+
+  if (!validationErrors.isEmpty()) { 
+    const errors = {};
+    
+    validationErrors
+      .array()
+      //@ts-ignore
+      .forEach(error => errors[error.path] = error.msg);
```

Comment:
we should make a global declaration to explain what these fields are supposed to be, for now ts-ignore and move on